### DISCUSSION TITLE
请问STM32F405RGT6可以使用STM32F407VET6替代吗?

### DISCUSSION THREAD

**Discussion by whlook** [peng-zhihui/ElectronBot#106] [Q&A]
如题，目前手里有407的芯片，我看硬件上是兼容的，软件上除了启动部分，其他有需要魔改(工作量较大)的地方吗，请了解的大佬解惑，感谢 :pray:

**Comment by leazer** [comment]
405RG 和 407VE 引脚数量不同吧。。一个64pin 一个100pin

**Comment by netube99** [comment]
封装引脚数都不一样，VE的体积大了点，你不好改吧？

**Comment by whlook** [comment]
抱歉，封装看错了 :rofl:

**Comment by xiaodou886** [comment]
选用STM32F407RGT6，起码封装和引脚定义得一样
